iml-system-docker-tests/tests/ldiskfs_test.rs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2020 DDN. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+mod utils;
```

Comment:
I don't think we need this

Reply:
How do I access the utils functions if I take out the mod? I tried taking it out and it won't let me use them.

Reply:
For visibility, I referenced https://doc.rust-lang.org/book/ch11-03-test-organization.html#submodules-in-integration-tests when making this change.